setup.py

```diff
@@ -126,7 +126,7 @@ def run(self):
 
         "psutil == 2.1.2",
         "netifaces >= 0.8",
-        "ipaddr == 2.1.10",
+        "ipaddr == 2.1.11",
```

Comment:
A concern I have is about what happens if a package stops being served by PyPI while we depend on it. I suggested serving our own packages.